ecs-cli/modules/cli/configure/configure.go

```diff
@@ -29,6 +33,67 @@ func fieldEmpty(field string, flagName string) error {
 	return nil
 }
 
+// Migrate is the callback for the Migrate command
```

Comment:
nit: Can you add more meaningful comments like "Migrate is to convert the old ini config to the new yaml config."?